pkg/cmd/cli/cmd/edit.go

```diff
@@ -286,7 +287,7 @@ func (h *editHeader) WriteTo(w io.Writer) error {
 		}
 		fmt.Fprintln(w, "#")
 	}
-	return nil
```

Comment:
Hmm… the point of this is to comply with a common interface. The returned number should be the number of bytes written, I think